ui/component/textareaWithSuggestions/view.jsx

```diff
@@ -0,0 +1,387 @@
+
+  React.useEffect(() => {
+    if (!stringifiedResults) return;
+
+    const arrayResults = JSON.parse(stringifiedResults);
+    if (arrayResults && arrayResults.length > 0) {
+      doResolveUris(arrayResults);
+      doSetSearchResults(arrayResults);
+    }
+  }, [doResolveUris, doSetSearchResults, stringifiedResults]);
+
+  // Disable sending on Enter on Livestream chat
+  React.useEffect(() => {
+    if (!isLivestream) return;
+
+    if (suggestionTerm && inputRef) {
+      inputRef.current.setAttribute('term', suggestionTerm);
+    } else {
+      inputRef.current.removeAttribute('term');
+    }
+  }, [inputRef, isLivestream, suggestionTerm]);
+
+  // Only resolve commentors on Livestreams when first trying to mention/looking for it
+  React.useEffect(() => {
+    if (isLivestream && commentorUris && suggestionTerm) doResolveUris(commentorUris);
+  }, [commentorUris, doResolveUris, isLivestream, suggestionTerm]);
+
+  // Allow selecting with TAB key
+  React.useEffect(() => {
+    if (!suggestionTerm) return; // only if there is a term, or else can't tab to navigate page
+
+    function handleKeyDown(e: SyntheticKeyboardEvent<*>) {
+      const { keyCode } = e;
+
+      if (highlightedSuggestion && keyCode === KEYCODES.TAB) {
+        e.preventDefault();
+        handleSelect(highlightedSuggestion.label);
+      }
+    }
+
+    window.addEventListener('keydown', handleKeyDown);
+
+    return () => window.removeEventListener('keydown', handleKeyDown);
+  }, [handleSelect, highlightedSuggestion, suggestionTerm]);
+
+  /** ------ **/
+  /** Render **/
+  /** ------ **/
+
+  const renderGroup = (group: string, children: any) => (
+    <div className="textareaSuggestions__group">
+      <label className="textareaSuggestions__label">
+        {group}
+        {suggestionTerm && suggestionTerm.length > 1
+          ? ' ' + __('matching %matching_term%', { matching_term: suggestionTerm })
+          : ''}
+      </label>
+      {children}
+      <hr className="textareaSuggestions__topSeparator" />
+    </div>
+  );
+
+  const renderInput = (params: any) => {
+    const { InputProps, disabled, fullWidth, id, inputProps: autocompleteInputProps } = params;
+    const inputProps = { ...autocompleteInputProps, ...inputDefaultProps };
+    const autocompleteProps = { InputProps, disabled, fullWidth, id, inputProps };
+
+    return <TextField inputRef={inputRef} multiline select={false} {...autocompleteProps} />;
+  };
+
+  const renderOption = (optionProps: any, label: string) => {
+    const emoteFound = isEmote && EMOTES.find(({ name }) => name.toLowerCase() === label);
+    const emoteValue = emoteFound ? { name: label, url: emoteFound.url } : undefined;
+    const emojiFound = isEmote && EMOJIS.getUnicode(label);
+    const emojiValue = emojiFound ? { name: label, unicode: emojiFound } : undefined;
+
+    return <TextareaSuggestionsItem uri={label} emote={emoteValue || emojiValue} {...optionProps} />;
+  };
+
+  return (
+    <Autocomplete
```

Comment:
After typing `@` in a comment text field.  Just need to put a key for each item in the list.

![image](https://user-images.githubusercontent.com/64950861/144954845-c33d24fc-c383-4d71-b11b-1f523e09dcfe.png)